GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2976,7 +2978,14 @@ private void FillTerminalTab()
                     }
                 }
 
-                _terminal.Start(startInfo, ThreadHelper.JoinableTaskFactory);
+                try
+                {
+                    _terminal.Start(startInfo, ThreadHelper.JoinableTaskFactory);
+                }
+                catch (InvalidOperationException)
+                {
+                    MessageBox.Show(_comEmuNotFound.Text, _errorCaption.Text, MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
In addition to the above, I see no reason to show the message box for Release builds, nor translate the message to our users. I want the app to crash, if a user has decided to randomly delete files.

To this extent, I suggest we do the following:
```suggestion
#if DEBUG
                    MessageBox.Show(@"ConEmu appears to be missing. Please perform a full rebuild and try again.", @"Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
#else
                    throw;
#endif
```

Because this affects only dev time experience, there is no need to translate the message, and the string must be inlined

Reply:
I agree that the translation is not needed if it's for developper only. I will do another commit in few hours.